EndlessClient/Controllers/InventoryController.cs

```diff
@@ -50,6 +51,9 @@ public class InventoryController : IInventoryController
         private readonly IEOMessageBoxFactory _eoMessageBoxFactory;
         private readonly IChatRepository _chatRepository;
         private readonly ILocalizedStringFinder _localizedStringFinder;
+        private readonly ISfxPlayer _sfxPlayer;
+
+        private bool _goldWarningShown = false;
```

Comment:
Now that this class contains state, you will need to change `[AutoMappedType]` to `[AutoMappedType(IsSingleton = true)]`

Reply:
![image](https://github.com/ethanmoffat/EndlessClient/assets/70440468/62faa6a2-7376-4d1c-9843-7ed331274e8d)